# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/performance/browser_preferences_usage.js
# Commit: b55aacf910e7
# Full Hash: b55aacf910e704508a5820c09b25411142ccf7cb
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2024-06-25 09:36:02
# Regressor Bug: 1728331
# File Overlap Count: 1
# Description:
#   Bug 1728331 - Part 4: Make ContentParent KeepAlives explicit with RAII references, r=smaug,dom-worker-reviewers,asuth
#   
#   This is a fairly significant patch, however it would be difficult to break it
#   down into smaller patches:
#   
# ==============================================================================

diff -r 3dd07ed49638 -r b55aacf910e7 browser/base/content/test/performance/browser_preferences_usage.js
--- a/browser/base/content/test/performance/browser_preferences_usage.js	Mon Jun 24 23:19:28 2024 +0000
+++ b/browser/base/content/test/performance/browser_preferences_usage.js	Mon Jun 24 23:19:28 2024 +0000
@@ -186,8 +186,8 @@
     // This pref is only accessed in automation to speed up tests.
     knownProblematicPrefs["dom.ipc.keepProcessesAlive.webIsolated.perOrigin"] =
       {
-        min: 100,
-        max: 102,
+        min: 50,
+        max: 51,
       };
     if (AppConstants.platform == "linux") {
       // The following sandbox pref is covered by